src/LabMidiSong.cpp

```diff
@@ -445,7 +446,7 @@ MidiEvent* parseEvent(uint8_t const*& dataStart, uint8_t lastEventTypeByte)
                 auto event = new Event_TimeSignature();
                 double num = double(*dataStart++);
                 double denom = double(*dataStart++);
-                event->timeSignature = num / std::powf(2., denom);
+                event->timeSignature = num / std::pow(2., denom);
```

Comment:
this is probably the fix here, right? <cmath> defines std::pow, but math.h defines plain ol' pow, so the math.h include is unnecessary?